charliechauri/charliechauri
# feat: theme selector and dark theme

# Introduce Theme Selector

## 📖 Description/Context

Add theme selector, currently supporting light (default) and dark
- Use Context API for powering theme
- Introduce `<GlobalSelectors />` for grouping theme selector and language selector
- Introduce `<Link />` for supporting dark theme

## Trello/Jira/Etc

Fixes: #39, #5 

## Screenshots

![themes_support](https://user-images.githubusercontent.com/11186658/84604105-491a3c80-ae59-11ea-933e-ed06720506ff.gif)


## 🧪 How to test

1. Run `yarn start`
1. Go to [homepage](http://localhost:3000/)
1. Default theme is light
1. Click on theme selector
1. Select dark
1. Theme should change
1. Click on theme selector again
1.  Select light
1. Should display default light theme
1. ✨
